docs/platforms/javascript/common/configuration/micro-frontend-support.mdx

```diff
@@ -37,61 +46,68 @@ module.exports = {
     sentryWebpackPlugin({
       /* Other plugin config */
       _experiments: {
-        moduleMetadata: ({ org, project, release }) => {
-          return { team: "frontend", release };
-        },
+        moduleMetadata: ({ release }) => ({ dsn: "__MODULE_DSN__", release }),
       },
     }),
   ],
 };
 ```
 
-### `ModuleMetadata` Integration
-
-Requires SDK version `7.59.0` or higher.
-
 Once metadata has been injected into modules, the `moduleMetadataIntegration` 
 can be used to look up that metadata and attach it to stack frames with
-matching file names. This metadata is then available in other integrations or in
-the `beforeSend` callback as the `module_metadata` property on each
-`StackFrame`. This can be used to identify which bundles may be responsible
-for an error and used to tag or route events.
+matching file names. This metadata is then available in the `beforeSend` callback
+as the `module_metadata` property on each `StackFrame`. This can be used to identify
+which bundles may be responsible for an error and, once the destination is determined,
+store it as a list of DSN-release pairs in `event.extra[MULTIPLEXED_TRANSPORT_EXTRA_KEY]` 
+where multiplexed transport knows to look for.
```

Comment:
```suggestion
as the `module_metadata` property on each `StackFrame`. This can be used to identify
which bundles may be responsible for an error. Once the destination is determined, you can
store it as a list of DSN-release pairs in `event.extra[MULTIPLEXED_TRANSPORT_EXTRA_KEY]`
for the multiplexed transport to reference for routing.
```